kiwix-build.py

```diff
@@ -399,7 +400,14 @@ def download(self, what, where=None):
             if what.sha256 == get_sha256(file_path):
                 raise SkipCommand()
             os.remove(file_path)
-        urllib.request.urlretrieve(file_url, file_path)
+        if options.no_cert_check == True:
+            context = ssl.create_default_context()
+            context.check_hostname = False
+            context.verify_mode = ssl.CERT_NONE
+            with urllib.request.urlopen(file_url, context=context) as resource, open(file_path, 'wb') as file:
+                file.write(resource.read())
```

Comment:
Well maybe not create a **new** function just for this.
But the function we are changing is already a `download` function. We just have to add a loop around the `read` to make it works with small chunks.